src/machines/__tests__/auth-token-polling-machine.test.ts

```diff
@@ -80,27 +80,24 @@ test('it schedules the next poll, then done', (done) => {
 })
 
 test('polls several times for the value', (done) => {
-  const clock = new SimulatedClock()
-
   // mock a couple failed requests followed by a successful request
   const requestAuthToken = jest
     .fn()
-    .mockRejectedValueOnce(null)
-    .mockRejectedValueOnce(null)
-    .mockResolvedValueOnce({authToken: 'auth123'})
+    .mockRejectedValueOnce(null) // 1st request fails
+    .mockRejectedValueOnce(null) // 2nd request fails
+    .mockResolvedValueOnce({authToken: 'auth123'}) // 3rd request succeeds!
 
   const pollingService = interpret(
     authTokenPollingMachine.withConfig({
       services: {
         requestAuthToken,
       },
+      delays: {
+        WAIT_BETWEEN_POLLS: 0,
+      },
```

Comment:
Overriding the polling delay to `0ms` means the test can quickly simulate the machine's behavior without using the `SimulatedClock` to _advance_ time.